DittoExportLogs/src/main/java/live/ditto/dittoexportlogs/ExportLogs.kt

```diff
@@ -1,26 +1,97 @@
 package live.ditto.dittoexportlogs
 
+import android.os.Build
+import android.os.Environment
+import androidx.annotation.RequiresApi
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.width
+import androidx.compose.material3.AlertDialog
+import androidx.compose.material3.Button
+import androidx.compose.material3.CircularProgressIndicator
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
 import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.rememberCoroutineScope
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalContext
 import androidx.compose.ui.res.stringResource
-import live.ditto.exporter.ExportDialog
-import java.nio.file.Path
+import androidx.compose.ui.unit.dp
+import kotlinx.coroutines.launch
+import java.io.File
+import java.time.LocalDateTime
+import java.time.format.DateTimeFormatter
 
+@RequiresApi(Build.VERSION_CODES.O)
 @Composable
-fun ExportLogs(onDismiss: () -> Unit) {
-    ExportDialog(
-        title = stringResource(R.string.export_logs),
-        text = stringResource(R.string.do_you_want_to_export_logs),
-        confirmText = stringResource(R.string.export),
-        cancelText = stringResource(R.string.cancel),
-        fileProvider = { getZippedLogs().toFile() },
-        mimeType = stringResource(R.string.application_x_zip),
-        onDismiss = onDismiss
+fun ExportLogs(onDismiss: () -> Unit, viewModel: ExportLogsViewModel = ExportLogsViewModel()) {
+
+    val isLoading by remember { mutableStateOf(viewModel.isLoading)}
+
+    val title = stringResource(R.string.export_logs)
+
+    val confirmText = stringResource(R.string.export)
+    val cancelText = stringResource(R.string.cancel)
+    val modifier = Modifier
+    val coroutineScope = rememberCoroutineScope()
+    val context = LocalContext.current
+
+    val now = LocalDateTime.now().format(DateTimeFormatter.ofPattern("yyyyMMdd_HHmmss"))
+    val appName = context.applicationInfo.loadLabel(context.packageManager).toString()
+
+    val fileName = "$appName-dittologs-$now.jsonl.gz"
+    val downloadsDir = Environment.getExternalStoragePublicDirectory(Environment.DIRECTORY_DOWNLOADS);
+    val filePath = File(downloadsDir, fileName).absolutePath
+
+    AlertDialog(
+        modifier = modifier,
```

Comment:
We can delete this parameter here as we never change it. And then remove the `val modifier = Modifier` on line 39 above.